Pod/Classes/MenuView.swift

```diff
@@ -85,21 +85,21 @@ public class MenuView: UIScrollView {
     
     // MARK: - Lifecycle
     
-    internal init(menuItemTitles: [String], menuItemDescs: [String], options: PagingMenuOptions) {
+    public init(menuItemTitles: [String], menuItemDescs: [String], options: PagingMenuOptions) {
```

Comment:
It should be done soon, but I can merge the multiline text part only first!
Well, its safer for you to wait the feature though.
Sorry for my inconvenience.